pkg/features/pilot/pilot.go

```diff
@@ -117,24 +119,31 @@ var (
 	// EnableCDSPrecomputation provides an option to enable precomputation
 	// of CDS output for all namespaces at the start of a push cycle.
 	// While it reduces CPU, it comes at the cost of increased memory usage
+	_, _                    = env.GetVar("PILOT_ENABLE_CDS_PRECOMPUTATION", "", "")
 	EnableCDSPrecomputation = func() bool {
-		return len(os.Getenv("PILOT_ENABLE_CDS_PRECOMPUTATION")) != 0
+		enableCDSPrecomputation, _ := env.GetVar("PILOT_ENABLE_CDS_PRECOMPUTATION", "", "")
+		return len(enableCDSPrecomputation) != 0
```

Comment:
This would change the logic here, but should this be a standard bool? Super confusing that `PILOT_ENABLE_CDS_PRECOMPUTATION=false` turns this on

Reply:
Yeah, ideally these two should just be standard bools. But I didn't want to go chasing the uses down, including in docs, so I preserved the same semantics. Pilot folks can come clean this up if desired.